### ISSUE TITLE
Errored out while deleting default team[developers] in Organization

### ISSUE THREAD

**Issue by indcoder** [npm/www#169]
I wanted to change the default name of the team called developers in the organization . Unfortunately could not find a way to just edit the name , so I thought lets delete the organization . Unfortunately it errored and told me to give the below to you.

1493494345908:www-18-west.internal.npmjs.com:25479:j229jfjf:58281

**Comment by ashleygwilliams** [comment]
hi @indcoder ! do you mean that you tried to delete the team "developers"?

**Comment by indcoder** [comment]
Yes @ashleygwilliams . That error occured when I tried to delete the team developers. Funny thing: I logged in again and tried to delete it .Voila, it went through . My 2c : When I first tried to delete it , I think the login credentials had expired [because I came to the page after discussing with the team about the name] but instead of showing an authentication error, it crashed on carrying out the action and spewed the error code that I posted at the end of the issue detail
